packages/inter-protocol/src/price/priceAggregatorChainlink.js

```diff
@@ -219,10 +219,14 @@ export const start = async (zcf, privateArgs, baggage) => {
         const invitationMakers = Far('invitation makers', {
           /** @param {import('./roundsManager.js').PriceRound} result */
           PushPrice(result) {
-            return zcf.makeInvitation(cSeat => {
-              cSeat.exit();
-              admin.pushPrice(result);
-            }, 'PushPrice');
+            return zcf.makeInvitation(
+              /** @param {ZCFSeat} cSeat */
+              async cSeat => {
+                cSeat.exit();
+                await admin.pushPrice(result);
```

Comment:
when this throws, `getOfferResult()` rejects

Reply:
that seems right. (Sorry if I misled you previously.) 

`getOfferResult()` is supposed to give you the return value of the offerHandler, so if it throws, `getOfferResult()` has nothing else to return.  The exitSubscriber should still report that the seat has exited, and numWantsSatisfied should be 1.